.github/CODEOWNERS

```diff
@@ -146,26 +146,29 @@
 
 # Dev workflow team.
 /.circleci/ @U8NWXD @kevintab95
-/.eslintignore @DubeySandeep
-/.eslintrc @DubeySandeep
-/.htmllintrc @DubeySandeep
-/.coveragerc @DubeySandeep
+/.eslintignore @Hudda @DubeySandeep
+/.eslintrc @Hudda @DubeySandeep
+/.htmllintrc @Hudda @DubeySandeep
+/.coveragerc @Hudda @DubeySandeep
 /.gitattributes @DubeySandeep
 /.gitignore @DubeySandeep
-/.isort.cfg @DubeySandeep
-/.pylintrc @DubeySandeep
-/.stylelintrc @DubeySandeep
+/.isort.cfg @Hudda @DubeySandeep
+/.pylintrc @Hudda @DubeySandeep
+/.stylelintrc @Hudda @DubeySandeep
 /.yarnrc @DubeySandeep
 /codecov.yml @nithusha21
-/core/templates/css/.stylelintrc @DubeySandeep
-/tox.ini @DubeySandeep
+/core/templates/css/.stylelintrc @Hudda @DubeySandeep
+/tox.ini @Hudda @DubeySandeep
 /scripts/*.py @DubeySandeep
 /scripts/check_e2e_tests_are_captured_in_ci*.py @kevintab95 @DubeySandeep
 /scripts/check_frontend_test_coverage*.py @DubeySandeep @nithusha21
 /scripts/create_topological_sort_of_all_services*.py @DubeySandeep
 /scripts/install_prerequisites.sh @DubeySandeep
 /scripts/typescript_checks*.py @DubeySandeep @vojtechjelinek
-/scripts/linters/ @DubeySandeep
+/scripts/linters/ @Hudda @DubeySandeep
+/scripts/linters/codeowner_linter*.py @sajalasati
+/scripts/linters/pylint_extensions*.py @sajalasati
+/scripts/linters/python_linter*.py @sajalasati
```

Comment:
Let us (me and Sajal) co-own these files initially and later Sajal can own this independently.

@sajalasati is that fine with you?

Reply:
Ohh sorry I should have probably clarified about this earlier. Since you already own `/scripts/linters folder` I thought you would always be asked for a codeowner review too - If this is not the case (meaning ownership for specific files do override the higher level ownerships), then yes my plan was to co-own these files with you for sometime, and would request @Hudda to make the change then. Again sorry for my misunderstanding.

Reply:
@Hudda, Can you please add me for the above files as well? :)

Reply:
Done.